Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs

```diff
@@ -140,7 +140,11 @@ private string EvaluateJavaScript(string nodeExePath, string testFile, string di
 
             var processStartInfo = new ProcessStartInfo(nodeExePath, arguments)
             {
+#if DEBUG
```

Comment:
Can you put a comment as to why different behavior here for debug or release builds is desired.